dom.bs

```diff
@@ -639,10 +639,18 @@ dictionary EventInit {
 <p>An {{Event}} object is simply named an <dfn export id=concept-event>event</dfn>. It allows for
 signaling that something has occurred, e.g., that an image has completed downloading.</p>
 
+<p>An <a>event</a> has an associated <dfn export for=Event>relatedTarget</dfn> (an
+{{EventTarget}} object). Unless stated otherwise it is null.</p>
+
+<p>An <a>event</a> has an associated <dfn export>set the relatedTarget</dfn> algorithm,
+which takes an {{EventTarget}} <var>relatedTarget</var>. Unless specified otherwise it does
+nothing.</p>
```

Comment:
Ah. I think we can.

- In DOM Standard, we use only "relatedTarget" concept.
- In UI Events, define relatedTarget attribute as reflecting "relatedTarget" concept.

That sounds reasonable. Let me try it.